tests/phpunit/tests/formatting/LikeEscape.php

```diff
@@ -10,21 +10,21 @@ class Tests_Formatting_LikeEscape extends WP_UnitTestCase {
 	 */
 	function test_like_escape() {
 
-		$inputs = array(
+		$inputs   = array(
 			'howdy%', //Single Percent
 			'howdy_', //Single Underscore
 			'howdy\\', //Single slash
 			'howdy\\howdy%howdy_', //The works
 		);
 		$expected = array(
-			"howdy\\%",
+			'howdy\\%',
 			'howdy\\_',
 			'howdy\\',
-			'howdy\\howdy\\%howdy\\_'
+			'howdy\\howdy\\%howdy\\_',
```

Comment:
Should these quotes be changed?

Reply:
Single quotes in use upstream:
https://github.com/WordPress/wordpress-develop/blob/trunk/tests/phpunit/tests/formatting/likeEscape.php

Reply:
I think upstream is broken.

Reply:
I think I was misreading it.  My mistake.